apps/worker/src/app/workflow/usecases/send-message/send-message-sms.usecase.ts

```diff
@@ -272,9 +272,9 @@ export class SendMessageSms extends SendMessageBase {
       }
 
       const result = await smsHandler.send({
-        to: phone,
+        to: overrides.to || phone,
         from: integration.credentials.from,
```

Comment:
I think we can also allow overriding the from here

Reply:
makes sense, added it